platforms/r3-corda-ent/charts/auth/templates/deployment.yaml

```diff
@@ -112,10 +112,10 @@ spec:
             #[TODO] : The JWT creation can be moved to PKI Job
             keytool -genkeypair -alias oauth-test-jwt -keyalg RSA -keypass password -keystore etc/jwt-store.jks -storepass password -dname "{{ .Values.authSubject }}"
           resources:
-            {{- toYaml .Values.config.pod.resources | nindent 12 }}
-          volumeMounts:
```

Comment:
why is the volumeMounts section removed

Reply:
I was replacing resources value and must have accidentally removed volume mounts